vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+	"functions": {
+		"api/index.go": {
+			"memory": 2048,
+			"maxDuration": 60
```

Comment:
Is this in seconds? Minutes?

Reply:
MB ( but the higher the mb they more CPU you get)